table/block_based_table_reader.cc

```diff
@@ -806,7 +806,7 @@ Status BlockBasedTable::Open(const ImmutableCFOptions& ioptions,
     if (s.ok()) {
       s = ReadProperties(meta_iter->value(), rep->file.get(),
                          prefetch_buffer.get(), rep->footer, rep->ioptions,
-                         &table_properties);
+                         &table_properties, false /* block_type_not_available */);
```

Comment:
I rethink about it. I probably gave a bad suggestion. The double negative is harder to understand. Maybe "block_type_available".

Reply:
how about `reset_block_type`?

Reply:
I can't understand what it means for `ReadProperties()`.

Reply:
in that case I prefer calling it `block_type_missing` or `compression_type_missing`